streams/src/main/java/org/apache/kafka/streams/kstream/KStream.java

```diff
@@ -530,7 +1033,7 @@ void to(final TopicNameExtractor<K, V> topicExtractor,
      *                 this.state = context.getStateStore("myTransformState");
      *                 // punctuate each 1000ms; can access this.state
      *                 // can emit as many new KeyValue pairs as required via this.context#forward()
-     *                 context.schedule(Duration.ofSeconds(1), PunctuationType.WALL_CLOCK_TIME, new Punctuator(..));
+     *                 context.schedule(1000, PunctuationType.WALL_CLOCK_TIME, new Punctuator(..));
```

Comment:
Why this change? The new API accepts `Duration` (old API that accepts a long is deprecated).